config/initializers/kaminari_config.rb

```diff
@@ -1,3 +1,3 @@
 Kaminari.configure do |config|
-  config.default_per_page = 20
+  config.default_per_page = 10
```

Comment:
I changed it for a better page visualisation after completed the design of the page.